spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -83,51 +87,79 @@ Spring Cloud Stream provides a number of abstractions and primitives that simpli
 
 === Application structure
 
-A Spring Cloud Stream application consists of a middleware-neutral core that uses a Spring Messaging-based API and Spring Integration components, and communicates with the outside world through input and output channels that are managed and injected into it by the framework. A Spring Cloud Stream abstraction named a binder provides the bridging between the channels and the external brokers. Different binder implementations exist for different types of middleware, such as Kafka, Rabbit MQ, Redis or Gemfire, and an extensible API allows you to write your own binder.
+A Spring Cloud Stream application consists of a middleware-neutral core that communicates with the outside world through input and output channels.
+The channels are managed and injected into it by the framework, and a `Binder` connects them to the external brokers.
+Different `Binder` implementations exist for different types of middleware, such as Kafka, Rabbit MQ, Redis or Gemfire, and an extensible API allows you to write your own `Binder`.
 
 .Spring Cloud Stream Application
 image::SCSt-with-binder.png[width=300,scaledwidth="50%"]
 
-Spring Cloud Stream uses Spring Boot for configuration, and the binder abstraction makes it possible for Spring Cloud Stream applications to be flexible in terms of how it connects to the middleware.
+Spring Cloud Stream uses Spring Boot for configuration, and the `Binder` makes it possible for Spring Cloud Stream applications to be flexible in terms of how it connects to the middleware.
+For example, deployers can dynamically choose the destinations that these channels connect to at runtime (e.g. Kafka topics or Rabbit MQ exchanges).
+This can be done through external configuration properties in any form that is supported by Spring Boot (application arguments, environment variables, `application.yml` files, etc).
+Taking the sink example from the previous section, providing the `spring.cloud.stream.bindings.input.destination=raw-sensor-data` property to the application will cause it to read from the `raw-sensor-data` Kafka topic, or from a queue bound to the `raw-sensor-data` exchange in Rabbit MQ.
```

Comment:
add link to section on binder properties and middleware specific binder configuration.